docs/developing/install.rst

```diff
@@ -46,14 +46,13 @@ Install node by following the
 (the version of the nodejs package in the standard Ubuntu repositories is too
 old).
 
-Upgrade pip and npm, and install tox:
+Upgrade pip and npm, and install tox and tox-pip-extensions:
 
 .. code-block:: bash
 
-    sudo pip install -U pip tox
+    sudo pip install -U pip tox tox-pip-extensions
```

Comment:
This command does not specify the tox version, so the user will end up with the incompatible tox 3.5.0 by default.

Reply:
Fixed